backend/api/prompts.py

```
@@ -0,0 +1,120 @@
+async def test_prompt(
+    data: PromptTestRequest,
+    db: AsyncSession = Depends(get_db),
+    user_id: str = Depends(get_current_user)
+):
+    # Substitute variables
+    prompt_text = data.content
+    for k, v in data.variables.items():
+        prompt_text = prompt_text.replace(f"{{{{{k}}}}}", v)
+        
+    # Find active provider
+    provider_result = await db.execute(select(LLMProvider).where(LLMProvider.is_active == True))
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `LLMProvider.is_active:` for truth checks

Replace with `LLMProvider.is_active`

(E712)